.buildkite/pipeline.yml

```
@@ -171,6 +171,42 @@ steps:
       JULIA_PROJECT: "@amdgpu"
       JULIA_NUM_THREADS: 4
 
+  - label: "OpenCL Julia {{matrix.version}}"
+    matrix:
+      setup:
+        version:
+          - "1.10"
+          - "1.11"
+    plugins:
+      - JuliaCI/julia#v1:
+          version: "{{matrix.version}}"
+      - JuliaCI/julia-coverage#v1:
+          codecov: true
+    command: |
+      julia -e 'println("--- :julia: Developing OpenCL")
+                using Pkg
+                Pkg.develop(; name="OpenCL")
+                Pkg.develop(; name="SPIRVIntrinsics")
+                Pkg.develop(; name="GPUArrays")''
```

```suggestion
                Pkg.develop(; name="GPUArrays")'
```
Is this extra ' what's causing the test failures?